cobigen/cobigen-core-api/src/main/java/com/devonfw/cobigen/api/to/VariableAssignmentTo.java

```diff
@@ -21,7 +21,28 @@
   protected String value;
 
   /**
-   * Constructor to create a {@link VariableAssignmentTo} for a concrete string value
+   * True if the value is required, false if not
+   */
+  protected boolean mandatory;
+
+  /**
+   * Constructor to create a {@link VariableAssignment} for a concrete string value. Requiredness as boolean.
+   *
+   * @param type Type of the variable assignment, interpreted by the plug-ins
+   * @param varName variable name
+   * @param value concrete string value
+   * @author mbrunnli (15.04.2013)
```

Comment:
change author and date?

Reply:
We can even remove it. It's legacy from old code formatting